### PR TITLE
OSDOCS-7651: Moving MCO info 4.12

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-7651

Link to docs preview:
https://85977--ocpdocs-pr.netlify.app/openshift-enterprise/latest/architecture/control-plane.html
https://85977--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/machine-configuration-tasks.html

QE review:
- [x] SME approval

Additional information:

### CODE REVIEW

File: architecture/control-plane.adoc [hunk 1]
```diff
@@ -44,8 +44,6 @@ include::modules/arch-platform-operators.adoc[leveloffset=+2]
 * xref:../installing/overview/cluster-capabilities.adoc#cluster-capabilities[Cluster capabilities]
 endif::[]
 
-include::modules/understanding-machine-config-operator.adoc[leveloffset=+1]
-
```
Comment (mburke5678) [inline architecture/control-plane.adoc:48]: We should remove the additional references or move them to the new location.

**Comment by mburke5678** [comment]
@obrown1205 LGTM!

File: architecture/control-plane.adoc (hunk 1 above)
Comment (wking) [reply architecture/control-plane.adoc:48]: Also up around line 15, these seem like they're not core control-plane issues:

```
include::modules/architecture-machine-config-pools.adoc[leveloffset=+1]
[role="_additional-resources"]
.Additional resources
* xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-drift-detection_post-install-machine-configuration-tasks[Understanding configuration drift detection].

include::modules/architecture-machine-roles.adoc[leveloffset=+1]
```

Comment (obrown1205) [reply architecture/control-plane.adoc:48]: @mburke5678 WDYT?

Comment (mburke5678) [reply architecture/control-plane.adoc:48]: @wking The link to _Understanding configuration drift detection_ is related to the previous paragraph, where we introduce the concept of config drift. As for the inclusion of architecture-machine-roles.adoc, it appears that was [added 5 years ago](https://github.com/openshift/openshift-docs/commit/630b9a5a313eae683d94ec69996bb7fa059e749c#diff-dfa72e2ab79511e5580b4ad769004040f37752d8bbcb8952db6b2e9c0f57914a). Not sure if @kalexand-rh  would remember why it was added. 
Can we pursue this through a separate issue/PR, so we can move this one along?

Comment (obrown1205) [reply architecture/control-plane.adoc:48]: @wking already let me know in a DM yesterday that he was fine if we progressed on this one and had a separate PR for this. :) I'll mark it as SME-approved and pass it along for peer review. TYSM! @mburke5678 can you create the ticket for the follow-up?

Comment (mburke5678) [reply architecture/control-plane.adoc:48]: Follow up: https://issues.redhat.com/browse/OCPBUGS-45992

File: modules/machine-config-overview.adoc [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="machine-config-overview-{context}"]
-= Machine config overview
+= Machine Config overview
 
 The Machine Config Operator (MCO) manages updates to systemd, CRI-O and Kubelet, the kernel, Network Manager and other system features. It also offers a `MachineConfig` CRD that can write configuration files onto the host (see link:https://github.com/openshift/machine-config-operator#machine-config-operator[machine-config-operator]). Understanding what MCO does and how it interacts with other components is critical to making advanced, system-level changes to an {product-title} cluster. Here are some things you should know about MCO, machine configs, and how they are used:
```
Comment (lahinson) [inline modules/machine-config-overview.adoc:10]:
```suggestion
The Machine Config Operator (MCO) manages updates to systemd, CRI-O and Kubelet, the kernel, Network Manager and other system features. It also offers a `MachineConfig` CRD that can write configuration files onto the host (see link:https://github.com/openshift/machine-config-operator#machine-config-operator[machine-config-operator]). Understanding what the MCO does and how it interacts with other components is critical to making advanced, system-level changes to an {product-title} cluster. Here are some things you should know about the MCO, machine configs, and how they are used:
```
Does "MCO" take an article? In the first sentence, it looks like it does.

File: post_installation_configuration/machine-configuration-tasks.adoc [hunk 3]
```diff
@@ -27,6 +25,13 @@ include::modules/machine-config-node-drain.adoc[leveloffset=+2]
 * xref:../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-disabling-autoreboot-mco_troubleshooting-operator-issues[Disabling the Machine Config Operator from automatically rebooting]
 
 include::modules/machine-config-drift-detection.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* For more information about detecting configuration drift, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-drift-detection_post-install-machine-configuration-tasks[Understanding configuration drift detection].
+
+* For information about preventing the control plane machines from rebooting after the Machine Config Operator makes changes to the machine configuration, see xref:../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-disabling-autoreboot-mco_troubleshooting-operator-issues[Disabling Machine Config Operator from automatically rebooting].
```
Comment (lahinson) [inline post_installation_configuration/machine-configuration-tasks.adoc:33]:
```suggestion
* xref:../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-disabling-autoreboot-mco_troubleshooting-operator-issues[Disabling Machine Config Operator from automatically rebooting]
```

File: post_installation_configuration/machine-configuration-tasks.adoc [hunk 4]
```diff
@@ -27,6 +25,13 @@ include::modules/machine-config-node-drain.adoc[leveloffset=+2]
 * xref:../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-disabling-autoreboot-mco_troubleshooting-operator-issues[Disabling the Machine Config Operator from automatically rebooting]
 
 include::modules/machine-config-drift-detection.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* For more information about detecting configuration drift, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-drift-detection_post-install-machine-configuration-tasks[Understanding configuration drift detection].
```
Comment (lahinson) [inline post_installation_configuration/machine-configuration-tasks.adoc:31]:
```suggestion
* xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-drift-detection_post-install-machine-configuration-tasks[Understanding configuration drift detection]
```
Do not include text outside of the square brackets in additional resources: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

**Review by lahinson — COMMENTED** [review]
Just a couple of minor things to check. Looks good overall!

**Pull request merged**